### PR TITLE
Added recipe for autopep8

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by jakirkham** [comment]
Should we add `pep8` too?

**Comment by pmlandwehr** [comment]
It would make sense...

**Comment by pmlandwehr** [comment]
Done.  See [#740](https://github.com/conda-forge/staged-recipes/pull/740)

File: recipes/autopep8/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - pep8 >=1.5.7
+
+test:
+  commands:
+    - autopep8 --help
+    - autopep8 --version
+
+about:
+  home: https://github.com/hhatto/autopep8
+  license: MIT License
```
Comment (ocefpaf) [inline recipes/autopep8/meta.yaml]: I usually remove the extra word License there as the field is already named "license." (No need to change to get this merged though!)

**Comment by ocefpaf** [comment]
@pmlandwehr waiting for you answer on https://github.com/conda-forge/staged-recipes/pull/740/files#r66709209 to merge this one.

**Pull request closed without merging**